Sources/Get/get().swift

```diff
@@ -17,9 +17,9 @@ import Utility
  - Throws: Error.InvalidDependencyGraph
  - Returns: The modules that this manifest requires building
 */
-public func get(manifest: Manifest) throws -> [Package] {
+public func get(manifest: Manifest, manifestParser: (path: String, url: String) throws -> Manifest) throws -> [Package] {
```

Comment:
I think `(path: String, url: String) throws -> Manifest` can be typealiased as its being repeated a few times